e2e-environment/Dockerfile

```diff
@@ -1,12 +1,12 @@
 ARG RAIDEN_VERSION="v1.1.1"
 ARG CONTRACTS_PACKAGE_VERSION="0.37.1"
 ARG CONTRACTS_VERSION="0.37.0"
-ARG SERVICES_VERSION="645e106d406b6fc3e11441c8e85c76d3aff300d3"
-ARG SYNAPSE_VERSION="1.10.1"
+ARG SERVICES_VERSION="v0.13.1"
+ARG SYNAPSE_VERSION="v1.19.1"
```

Comment:
Why not using the latest version? :thinking: 
Or is this just the same as for the Python client?

Reply:
I went for the versions that run in the service bundle (and so on mainnet): https://github.com/raiden-network/raiden-service-bundle/blob/master/BUILD_VERSIONS